pyvista/plotting/plotting.py

```diff
@@ -1554,6 +1564,27 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
 
         ##### Plot a single PyVista mesh #####
 
+        silhouette_params = dict(rcParams['silhouette'])
```

Comment:
Is this just another way to say `rcParams['silhouette'].copy()`?

Reply:
Yes! At first I was using `copy` but I swicthed to this one because I found this way was used already in the codebase.